### DISCUSSION TITLE
"See Fewer Recommendation" dialog hidden by sidebar

### DISCUSSION THREAD

**Discussion by preaction** [community/community#121706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When clicking the "Show less activity like this" link, the resulting dialog is half-hidden behind an element on some browser widths. Switching to full-screen mode on my laptop expands the feed element and un-hides the dialog, as well as making my browser window smaller (hitting the "single column" responsive breakpoint).

<img width="1379" alt="Screen Shot 2024-04-30 at 1 59 12 PM" src="https://github.com/community/community/assets/53707/7f9c30e3-ec44-4213-b911-7dd1bf75719a">

Additionally, the label of the checkbox is not clickable, so only clicking the box itself works. This only appears to happen on the pop-up from a feed item "Based on your topics". This does not appear to be the case on other pop-ups from "Show less activity like this", in which the checkbox labels are a darker color (on the dark theme).

<img width="730" alt="Screen Shot 2024-04-30 at 2 08 03 PM" src="https://github.com/community/community/assets/53707/a66b9a89-e487-46af-a474-5ca33d3a6a5c">
<img width="718" alt="Screen Shot 2024-04-30 at 2 07 52 PM" src="https://github.com/community/community/assets/53707/bb861475-7d14-40f8-8dc7-710d47733a08">

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
